cmd/utils/flags.go

```diff
@@ -1844,7 +1805,7 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 	}
 	// Read the value from the flag no matter if it's set or not.
 	cfg.Preimages = ctx.GlobalBool(CachePreimagesFlag.Name)
-	if cfg.NoPruning && !cfg.Preimages {
+	if true || cfg.NoPruning && !cfg.Preimages { // TODO: Quorum; force preimages for contract extension and dump of states compatibility, until a fix is found
```

Comment:
TODO?

Reply:
yes I need to create a ticket for that

Reply:
ticket already present, I updated it
https://app.zenhub.com/workspaces/team-goquorum-5f97103021b1e5001fc7acce/issues/consensys/quorum-board/180